tests/RequestParserTest.php

```diff
@@ -252,7 +252,7 @@ private function createPostWithContent()
         $data .= "Host: localhost:8080\r\n";
         $data .= "User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.7; rv:32.0) Gecko/20100101 Firefox/32.0\r\n";
         $data .= "Connection: close\r\n";
-        $data .= "Content-Type: application/x-www-form-urlencoded\r\n";
+        $data .= "Content-Type: application/x-www-form-urlencoded; charset=utf-8\r\n";
```

Comment:
@WyriHaximus  Working on this now (sorry for the delay); however, since we are effectively testing the same exact thing as `RequestParserTest::testShouldReceivePostInBody()` already does, would keeping the single test but creating a dataProvider for that test with multiple variations of headers work?